aemedge/blocks/hero/hero.css

```diff
@@ -87,11 +87,61 @@ udex-hero-banner .hero-banner--image-above-cta {
   }
 }
 
+udex-hero-banner .hero-banner .media-blend__tags {
+  margin: var(--udexSpacer20) 0;
+}
+
+udex-hero-banner .hero-banner .media-blend__tags a {
+  font-size: var(--udexTypographyBodyXSFontSize);
+  text-decoration: none;
+  text-align: center;
+  padding: var(--udexSpacer2) var(--udexSpacer8);
+  background-color: var(--udexColorGrey2);
+  color: var(--udexColorGrey9);
+  border-radius: var(--udexRadiusS);
+  border: 1px solid var(--udexColorGrey2);
+}
+
 udex-hero-banner .hero-banner .media-blend__intro-text {
   font-family: var(--sapFontFamily);
   font-size: var(--udexTypographyEyebrowMFontSize, 14px);
+  font-weight: var(--udexTypographyFontWeightMedium);
   text-transform: uppercase;
   color: var(--udexColorGrey6, #5b738b);
+  text-decoration: none;
+
+  .eyebrow-arrow {
+    display: inline-block;
+    height: 10px;
+    width: 6px;
+    margin: 0 var(--udexSpacer8);
+
+    /* Use mask in order to be able to adjust SVG colour */
+    mask: url("/aemedge/icons/slim-arrow-left.svg") center no-repeat;
+    background-color: currentcolor;
+  }
+}
+
+udex-hero-banner .hero-banner .media-blend__intro-text a {
+  color: inherit;
+  text-decoration: inherit;
+
+  &:visited {
+    color: var(--udexColorGrey9);
+  }
+
+  &:hover {
+    color: var(--udexColorGrey8);
+  }
+
+  &:active {
+    color: var(--udexColorGrey10);
+  }
+}
+
+udex-hero-banner .hero-banner .media-blend__tags a:hover {
```

Comment:
These anchor styles are in a strange order as the linter was complaining about descending specificity